omnisafe/adapter/offpolicy_adapter.py

```diff
@@ -208,7 +209,7 @@ def _reset_log(self, idx: int | None = None) -> None:
         """Reset log.
 
         Args:
-            idx (int | None): The index of the environment.
```

Comment:
Follow the PyTorch guideline for type annotations in docstrings ([Docstring type formatting](https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md#docstring-type-formatting)).